warehouse/models/mart/gtfs/fct_monthly_route_service_by_timeofday.sql

```diff
@@ -77,26 +89,26 @@ daypart_aggregations AS (
         SUM(service_hours) AS ttl_service_hours
 
     FROM service_with_daypart
-    GROUP BY 1, 2, 3, 4, 5, 6, 7, 8, 9, 10
+    GROUP BY 1, 2, 3, 4, 5, 6, 7, 8, 9
 ),
 
-fct_scheduled_service_by_daypart AS (
+fct_monthly_route_service_by_timeofday AS (
     SELECT
-
+       {{ dbt_utils.generate_surrogate_key(['source_record_id', 'route_id', 'month', 'year', 'time_of_day']) }} AS key,
         name,
         source_record_id,
         route_id,
         route_short_name,
         route_long_name,
         time_of_day,
-        hour,
         month,
-        year,
```

Comment:
Would it be helpful to keep `year` here? Otherwise it seems like the same months across different years would be aggregated.

Reply:
Oh nice catch, yes, definitely want to keep `year`.